README.md

```diff
@@ -1,5 +1,17 @@
 # <div align="left"><img src="https://rapids.ai/assets/images/rapids_logo.png" width="90px"/>&nbsp;RAFT: Reusable Accelerated Functions and Tools</div>
 
+[![Build Status](https://gpuci.gpuopenanalytics.com/job/rapidsai/job/gpuci/job/raft/job/branches/job/raft-branch-pipeline/badge/icon)](https://gpuci.gpuopenanalytics.com/job/rapidsai/job/gpuci/job/raft/job/branches/job/raft-branch-pipeline/)
+
+## Resources
+
+- [RAFT Reference Documentation](https://docs.rapids.ai/api/raft/stable/): API Documentation.
+- [Getting Started](https://rapids.ai/start.html): Instructions for installing RAFT.
```

Comment:
I think maybe we should remove this one or rename it to "RAPIDS Getting Started" here and provide links to the "RAFT getting started" and "Build and Install RAFT" instructions from the RAFT docs above it. 

Like RMM, RAFT isn't explicitly listed on the RAPIDS getting started page.

Reply:
I've updated to point to installing and getting-started sections in the README (making this section partially act like a TOC I guess).  

I couldn't link to the docs site just yet for the build + getting started though:   The stable 'build.html' page doesn't exist here yet https://docs.rapids.ai/api/raft/stable/build.html , though I do see it at on the nightly docs site here https://docs.rapids.ai/api/raft/nightly/build.html  

(alternatively, I could link to the https://docs.rapids.ai/api/raft/nightly pages for these two links and update once 22.12 is out the door - )

Reply:
What do you think about just lining to the markdown files for those two for 22.12?

Reply:
I think that sounds good for this release -